tests/data/props-output.js

```diff
@@ -85,8 +92,9 @@ export const propsOutput = {
 					'cardButtonContent',
 				],
 				notExpected: [
-					'blockquoteCardImageUrl',
+					'blockquoteCardImageUrl'
```

Comment:
```suggestion
					'blockquoteCardImageUrl',
```
Didn't we say that there should be trailing comma